docs/cmd/kn_subscription_create.md

```diff
@@ -0,0 +1,46 @@
+## kn subscription create
+
+Create a subscription
+
+### Synopsis
+
+Create a subscription
+
+```
+kn subscription create NAME
+```
+
+### Examples
+
+```
+
+  # Create a subscription 'sub0' from InMemoryChannel 'pipe0' to a subscriber ksvc 'receiver'
+  kn subscription create sub0 --channel imcv1beta1:pipe0 --subscriber ksvc:receiver
+
+  # Create a subscription 'sub1' from KafkaChannel 'k1' to ksvc 'mirror', reply to a broker 'nest' and DeadLetterSink to a ksvc 'bucket'
+  kn subscription create sub1 --channel messaging.knative.dev:v1alpha1:KafkaChannel:k1 --subscriber mirror --reply broker:nest --dead-letter-sink bucket
+```
+
+### Options
+
+```
+      --channel string            Specify the channel to subscribe to, in the format '--channel Group:Version:Kind:Name'. You can use channel type aliases from kn config with this flag. You can also refer inbuilt channel type InMemoryChannel using alias 'imc' like '--type imc:CHANNEL_NAME'. Examples: '--channel messaging.knative.dev:v1alpha1:KafkaChannel:k1' for specifying explicit Group:Version:Kind:Name.
```

Comment:
> Is there are a way to reference a channel only by its name ? I.e. is there kind of a "default" channel type that would be picked when no GVK coordinates are specified.

No. We expect to have an explicit channel type mentioned.

(for channel CRUD, we defer the type of channel when creating to messaging layer default, and require only name of channel to create)

Reply:
> However I wonder whether we can already come up with a default set of mappings that are already covered by eventing, like the kafka broker that you mention here.

sure, this came up during the channel CRUD implementation as well, but since kafka source isnt a native install of eventing, we didnt assume the GVK of same to map to a default alias.

Reply:
yeah, I guess also that we then again have issues with the proper version. E.g. I would reserve `kafka:` for `v1` (or the "latest") available version (so that the standard prefix can grow with the version)